### DISCUSSION TITLE
Setting up a _post in a docs/ folder of a project - site not found

### DISCUSSION THREAD

**Discussion by frankieliu** [community/community#21555] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,</p>
<p>I am having problems in getting github pages to correct process the _post directory for a project.</p>
<p>Setup:</p>
<p>I have added a docs/ folder to my project frankieliu/pgm and I initialized it via</p>
<p>jekyll build</p>
<p>I have also installed github-pages gem via gem install github-pages, and modified the Gemfile with</p>
<p>gem “github-pages”, group: :jekyll_plugins</p>
<p>I tested with </p>
<p>bundle exec jekyll serve</p>
<p>And I can navigate to the posts through the browser</p>
<p>I commited and pushed these changes, and tried to navigate to the corresponding project page and docs</p>
<p><a href="http://frankieliu.github.io/pgm" class="onebox" target="_blank" rel="nofollow noopener">frankieliu.github.io/pgm</a></p>
<p>I can see the index.html which was generated from index.md but when clicking through to the posts I get bad links: sorry this page does not exist.  Note I can see the links as  Github’s jekyll plugin is getting the correct files from _posts directory, but for some reason the links:</p>
<p><a href="https://frankieliu.github.io/jekyll/update/2017/11/02/welcome-to-jekyll.html" rel="nofollow noopener">https://frankieliu.github.io/jekyll/update/2017/11/02/welcome-to-jekyll.html</a></p>
<p><a href="https://frankieliu.github.io/github-pages/2017/11/02/setting-up-github-pages.html" rel="nofollow noopener">https://frankieliu.github.io/github-pages/2017/11/02/setting-up-github-pages.html</a></p>
<p>are not reachable.  I get an error “Sorry this page does not exist.”.  Note I get no such errors from the local server.</p>
<p>Could you please advice how to fix this problem?</p>
<p>Thanks,</p>
<p>-f</p>

**Comment by birdcar** [comment, marked as answer]
<p>Based on the links you posted, it looks like you may need to add a baseurl of “/pgm” to your “_config.yml” file. Baseurl can be a bit confusing, but you’ll generally want set a baseurl when you’re building a site that won’t be published at the root of the domain. Parker, a Jekyll maintainer, wrote an article on his personal blog that goes into more detail if you’re interested.</p>
<p><a href="https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/" rel="nofollow noopener">https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/</a></p>
<p>Having said all of that, I took a look at <a href="https://github.com/frankieliu/pgm" rel="nofollow noopener">your repository</a> and it appears that you’ve <a href="https://github.com/frankieliu/pgm/commit/817cd96c126e3f09a515a34eaf014e84ddbb72be" rel="nofollow noopener">figured this one out</a> on your own. Great work! :smiley:</p>
